utils/utils.py

```diff
@@ -707,7 +707,7 @@ def ckpt_is_compatible(in_ckpt_path: str, download_dir: str = None):
         isinstance(checkpoint["hyper_parameters"]["model"]["_target_"], str)
         isinstance(checkpoint["hyper_parameters"]["dataset"]["bands"], list)
         isinstance(checkpoint["hyper_parameters"]["dataset"]["classes_dict"], (dict, DictConfig))
-        return True
+        return True, checkpoint
```

Comment:
Why modify this return statement? This utility seems unused with your modifs. Did I miss something?